bokeh/core/has_props.py

```diff
@@ -258,9 +257,9 @@ def __setattr__(self, name, value):
             return
 
         props = sorted(self.properties())
-        deprecated = getattr(self, '__deprecated_attributes__', [])
+        descriptor = getattr(self.__class__, name, None)
```

Comment:
It might be nice to make `lookup` return `None` if there is no descriptor and use that: https://github.com/bokeh/bokeh/blob/810c35d059d4d3fac03a1a6f1f1254d2fc7ccf49/bokeh/core/has_props.py#L391

Reply:
This may affect how other code uses `lookup()`, so I wouldn't proceed with this in this PR.